yt_dlp/extractor/soundcloud.py

```diff
@@ -112,21 +111,29 @@ def _download_json(self, *args, **kwargs):
     def _initialize_pre_login(self):
         self._CLIENT_ID = self.cache.load('soundcloud', 'client_id') or 'a3e059563d7fd3372b49b37f00a00bcf'
 
+    def _verify_oauth_token(self, token):
+        if self._request_webpage(
+                self._API_VERIFY_AUTH_TOKEN % (self._API_AUTH_QUERY_TEMPLATE % self._CLIENT_ID),
+                None, note='Verifying login token...', fatal=False,
+                data=json.dumps({'session': {'access_token': token}}).encode()):
+            self._HEADERS = {'Authorization': f'OAuth {token}'}
```

Comment:
This will re-trigger verification for each subclass. Is that the intention? Otherwise, making `_HEADERS` a class variable and `self._HEADERS['Authorization'] = f'OAuth {token}'` will share it.

Reply:
Ah true. I don't wanna do that. But also not sure what you mean w/ your suggestion

Reply:
nvm figured it out

Reply:
```suggestion
            self._HEADERS['Authorization'] = f'OAuth {token}'
```